File: java/LinkedListApp.java
public class LinkedListApp {
	public static void main(String[] args) {
		LinkedList<Integer> linkedList = new LinkedList<Integer>();

		for (int i = 10; i > 0; i--){
			linkedList.addFirst(i);
		}

		System.out.println("Printing List after adding 1-10");
		linkedList.displayLinkedList();

		linkedList.addLast(17);
		System.out.println("Add 17 to end of the list");
		linkedList.displayLinkedList();

		System.out.println("Remove the first element in the list");
		linkedList.removeFirst();
		linkedList.displayLinkedList();
	}
}

File: LinkedList/kotlin/LinkedListApp.kt
fun main(args : Array<String>) {
	val linkedList = LinkedList<Int>()

	linkedList.removeFirst()
	for (i in 10 downTo 1) {
		linkedList.addFirst(i.toInt())
	}

	println("Added numbers 10-1 to start of list")
	linkedList.displayList()

	println("Added 17 to end of list")
	linkedList.addLast(17.toInt())
	linkedList.displayList()

	println("removeFirst from list")
	linkedList.removeFirst()
	linkedList.displayList()

}
